client/src/app/App.js

```diff
@@ -863,7 +863,7 @@ export class App extends PureComponent {
     const linter = tabProvider.getLinter(tab);
 
     if (!linter) {
-      return tab;
+      return;
```

Comment:
This makes sense.

Reply:
Yeah, this was a leftover from when the method always returned the tab. This behavior was changed recently.